examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<p>From a quick look with a color blindness simulator, it looks like this plot is not very easy to interpret for people with tritanomaly. I think we should use the <code>arviz-darkgrid</code>  theme and use <code>C0</code>  and <code>C1</code>  as colors here. The code to use this theme is also at https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>